demo/index.html

```diff
@@ -35,6 +35,9 @@
 				max-width: 500px;
 				margin: 0 auto;
 			}
+			.ML__keyboard {
+				left: 0;
+			}
```

Comment:
Weird. Shouldn't this be injected together with other styles?

https://github.com/arnog/mathlive/blob/98b40d34674e6bc3f6194ebf2f1899b0ca0cd0da/css/virtual-keyboard.less#L54

https://github.com/arnog/mathlive/blob/e9022a2fa04c95060cfa48bb77ffa4a20d28b2dd/src/editor/virtual-keyboard-utils.ts#L339-L341?

Reply:
I don't know. It is just for demo so let's ignore this. In Platform keyboard renders properly without those 3 lines.